Configure

```diff
@@ -1201,6 +1201,10 @@ foreach (keys %useradd) {
 # At this point, we can forget everything about %user and %useradd,
 # because it's now all been merged into the corresponding $config entry
 
+if (grep { $_ =~ /(?:^|\s)-static(?:\s|$)/ } @{$config{LDFLAGS}}) {
```

Comment:
Are you sure the regexp is needed?  After all, you're grepping a list of words...

Reply:
Admittedly, I was pulling in the regex from #10878. I will check without it.

Reply:
Looks to be not needed.